lib/src/model/nutrient_modifier.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:openfoodfacts/src/model/off_tagged.dart';
+
+enum NutrientModifier implements OffTagged {
+  approximately,
+  superiorTo,
+  inferiorTo;
+
+  @override
+  String get offTag => symbol;
+}
+
+extension NutrientModifierExtension on NutrientModifier {
```

Comment:
For the record we don't need an extension.
An `enum` can have `static`  methods.